src/components/Garden/Agreement/Agreement.js

```diff
@@ -35,7 +35,7 @@ function Agreement() {
   }
 
   return (
-    <LayoutGutter>
+    <LayoutGutter collapseWhenSmall>
```

Comment:
Not related with the sidebar but from @PJColombo video realized we needed to adjust the padding